signals/weather.py

```
@@ -0,0 +1,256 @@
+            )
+            new_alerts = enriched
+
+        # 5. Sort by severity
+        new_alerts.sort(
+            key=lambda a: _SEVERITY_ORDER.get(a.get("impact_level", "Low"), 4)
+        )
+
+        # 6. Clean up expired dedup entries (best-effort)
+        try:
+            cleanup_expired()
+        except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)